playbooks/roles/swag_ember/templates/environment.js

```diff
@@ -0,0 +1,77 @@
+/* jshint node: true */
+
+module.exports = function(environment) {
+  var ENV = {
+    modulePrefix: 'swag-ember',
+    environment: environment,
+    baseURL: '/',
+
+    locationType: 'auto',
+
+    contentSecurityPolicy: {
+      'frame-src': '\'self\' http://static.ak.facebook.com https://s-static.ak.facebook.com https://www.facebook.com',
+      'img-src': 'self https://www.facebook.com',
+      'style-src': '\'self\' \'unsafe-inline\'',
+      'script-src': '\'self\' http://connect.facebook.net https://graph.facebook.com https://www.facebook.com',
+      'connect-src': '\'self\' ws://localhost:35729 ws://0.0.0.0:35729 http://0.0.0.0:4200/csp-report http://localhost:8000 http://learninglocker.{{provision__fqdn}}/data/xAPI/'
+    },
+
+    EmberENV: {
+      FEATURES: {
+        // Here you can enable experimental features on an ember canary build
+        // e.g. 'with-controller': true
+      }
+    },
+
+    APP: {
```

Comment:
Not related to this commit per se, but I'm curious: last time I checked it looked like the ember version of swag wasn't set up to have nodes change appearance based on data from the LRS. Has that been added, and if not is it on your radar?